cli/command/defaultcontextstore.go

```diff
@@ -0,0 +1,182 @@
+package command
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/docker/cli/cli/config/configfile"
+	"github.com/docker/cli/cli/context/docker"
+	"github.com/docker/cli/cli/context/kubernetes"
+	"github.com/docker/cli/cli/context/store"
+	cliflags "github.com/docker/cli/cli/flags"
+	"github.com/docker/docker/pkg/homedir"
+	"github.com/pkg/errors"
+)
+
+const (
+	// DefaultContextName is the name reserved for the default context (config & env based)
+	DefaultContextName = "default"
+)
+
+// DefaultContext contains the default context data for all enpoints
+type DefaultContext struct {
+	Meta store.ContextMetadata
+	TLS  store.ContextTLSData
+}
+
+// DefaultContextResolver is a function which resolve the default context base on the configuration the env variables
```

Comment:
nit: I think something is missing at the end of the comment + resolve is missing an s?